addons_rksv/pos_rksv/static/src/js/screens.js

```diff
@@ -206,7 +211,17 @@ odoo.define('pos_rksv.screens', function (require) {
             this.events['click .export_crypt'] = 'export_crypt';
             this.events['click .start_receipt_set_valid'] = 'start_receipt_set_valid';
         },
+        willStart: function() {
```

Comment:
it does prevent the RKSV status Widget from getting startet, if rksv is enabled it will return a deferred which is resolved, when rksv is not enabled the deferred will never get resolved - so it will not start !